```sh
$ npm run chrome

> Elgg@ chrome D:\wamp\www\hackathon2
> node_modules/karma/bin/karma start js/tests/karma.conf.js --browsers Chrome

'node_modules' is not recognized as an internal or external command,
operable program or batch file.

npm ERR! Windows_NT 10.0.10586
npm ERR! argv "D:\\Program Files\\nodejs\\node.exe" "D:\\Program Files\\nodejs\\          node_modules\\npm\\bin\\npm-cli.js" "run" "chrome"
npm ERR! node v5.5.0
npm ERR! npm  v3.3.12
npm ERR! code ELIFECYCLE
npm ERR! Elgg@ chrome: `node_modules/karma/bin/karma start js/tests/karma.conf.j          s --browsers Chrome`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the Elgg@ chrome script 'node_modules/karma/bin/karma start j          s/tests/karma.conf.js --browsers Chrome'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the Elgg package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node_modules/karma/bin/karma start js/tests/karma.conf.js --browser          s Chrome
npm ERR! You can get their info via:
npm ERR!     npm owner ls Elgg
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     D:\wamp\www\hackathon2\npm-debug.log
```

--- a/package.json
+++ b/package.json
@@ -29,6 +29,7 @@
     "karma-phantomjs-launcher": "~0.1",
     "karma-requirejs": "~0.2",
     "load-grunt-config": "~0.16.0",
-    "phantomjs": "~1.9.7-14"
+    "phantomjs": "~1.9.7-14",
+    "requirejs": "~2.2.0"
   }
 }

